scripts/dockerbot/check_prs.py

```diff
@@ -277,9 +313,19 @@ def check_status(pr):
                 logger.info('Setting task {} for pr {} to {}'
                             .format(task['stream'], pr['number'], states[task['result']]))
                 set_status(commit, states[task['result']], task['stream'])
+            task_states[task['stream']] = states[task['result']]
```

Comment:
Ahhh that's the bit I was missing...........the line 154 bit. I wrote it but I didn't remember. I thought check_status was triggered for ALL PRs, thanks for clearing that up for me. Sorry for the delay on it.